Update index.md (#1946)
spinnaker/spinnaker.github.io: setup/features/notifications/index.md

Email in spinnaker is provided by [Spring Boot Mail
starter](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-email.html){:target="\_blank"}.

> ##### we listed popular email provider smtp settings below.

Reviewer: ```suggestion
The settings for some popular email providers are listed below:
```

> | Email Provider | SMTP username | SMTP password | SMTP server address | SMTP port (TLS) | SMTP port (SSL) | SMTP TLS/SSL required |

Reviewer 2: I Love the structure. A much needed change.
